keylime.conf

```diff
@@ -253,12 +241,6 @@ private_key = default
 # EK certificate storage location (directory).
 tpm_cert_store = /var/lib/keylime/tpm_cert_store/
 
-# Set the password needed to decrypt the private key file.
-# This should be set to a strong password.
-# if you are using the auto generated keys from the CV, set the same password
-# here as you did for 'private_key_pw' in the [cloud_verifier] section.
-private_key_pw = default
```

Comment:
We can't simply remove the password, there is validation at ca_utils, when it's None will not generate certifcates.
Tried '' as password, but haven't tested if it works.

Reply:
Understood. Can you give me more details on the failure?

Reply:
@maugustosilva the verification code is here:
https://github.com/keylime/keylime/blob/master/keylime/ca_util.py#L73
this will cause the cert generation fail.